nifi-toolkit/nifi-toolkit-assembly/docker/tests/exit-codes.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This file needs to be added to the excludes for the apache-rat-plugin in nifi-toolkit-assembly.